### PR TITLE
chore: now @remix-run/netliify-edge does not compile

### PR DESCRIPTION
**Pull request by nickytonline**
Closes: #

- [ ] Docs
- [ ] Tests

Testing Strategy:

### CODE REVIEW

File: packages/remix-netlify-edge/.empty.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*
```
Comment (nickytonline) [inline packages/remix-netlify-edge/.empty.js:1]: The deno package uses this file too so that rollup kicks in.

File: packages/remix-netlify-edge/package.json [hunk 2]
```diff
@@ -3,8 +3,7 @@
   "description": "Netlify Edge server request handler for Remix",
   "version": "1.8.0",
   "license": "MIT",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./index.ts",
```
Comment (nickytonline) [inline packages/remix-netlify-edge/package.json:6]: We're serving non-compiled TS now.

File: packages/remix-netlify-edge/package.json [hunk 3]
```diff
@@ -24,11 +24,5 @@
   },
   "engines": {
     "node": ">=14"
-  },
-  "files": [
```
Comment (nickytonline) [inline packages/remix-netlify-edge/package.json:28]: removed these as the deno package's package.json does not include a files property.

File: rollup.utils.js [hunk 4]
```diff
@@ -435,7 +435,7 @@ module.exports = {
 
 /**
  * @typedef {Record<string, { values?: string[]; types?: string[] }>} MagicExports
- * @typedef {"architect" | "cloudflare-pages" | "cloudflare-workers" | "express" | "netlify" | "netlify-edge" | "vercel"} RemixAdapter
+ * @typedef {"architect" | "cloudflare-pages" | "cloudflare-workers" | "express" | "netlify" | "vercel"} RemixAdapter
```
Comment (ascorbic) [inline rollup.utils.js:438]: Why did you remove this?

Comment (nickytonline) [reply rollup.utils.js:438]: Ahh, that was a mistake. I initially thought we didn't need it if we weren't compiling the package.
